### DISCUSSION TITLE
Mix different types of plot

### DISCUSSION THREAD

**Discussion by SMorettini** [alandefreitas/matplotplusplus#136] [Q&A]
Is it possible to mix two types of plots? 
For example, mixing a scatter plot and a line plot to show the results of a regression(as in the figure)?
![regression](https://user-images.githubusercontent.com/7838869/114692992-70f85c00-9d19-11eb-9e87-be8b68c85166.png)

**Comment by alandefreitas** [comment]
Sure. You just need to call the `hold` function on the axes: https://alandefreitas.github.io/matplotplusplus/appearance/multiplot/hold/
